agent/hcp/testing.go

```diff
@@ -141,7 +142,7 @@ func mockTokenResponse(w http.ResponseWriter) {
 }
 
 func (s *MockHCPServer) handleStatus(r *http.Request, cluster resource.Resource) (interface{}, error) {
-	var req gnmmod.HashicorpCloudGlobalNetworkManager20220215AgentPushServerStateRequest
+	var req hcpgnm.AgentPushServerStateBody
```

Comment:
these look the same to me but why change?

Reply:
Wasn't available when I updated the SDK, looks like it got deleted during this SDK update: https://github.com/hashicorp/hcp-sdk-go/commit/cc8c324b3c9aea3a9bbcc98a8c4360d419ff77b9#diff-05a7f927f805be79d9d9949aa4496eb989359ed21fb831f511a7590fe1103163L1